Test/server/counterexample_commandline.dfy.expect

```diff
@@ -4,21 +4,21 @@ counterexample_commandline.dfy(17,22): Related location: This is the postconditi
 Dafny program verifier finished with 1 verified, 1 error
 Counterexample for first failing assertion: 
 counterexample_commandline.dfy(20,8): initial state:
-	s : seq<char> = [?#0]
+	s : seq<char> = ['!']
```

Comment:
Do you have regressions that test other types, such as boolean, integer, etc.?
Or maybe that is not important for some reason I am maybe missing?

Reply:
There are XUnit tests for other types (`ArbitraryBitVector`, `ArbitraryBool`, etc. in [CounterExampleTest.cs](https://github.com/Dargones/dafny/blob/282b02ec488177875d2f20772cca90ba96f9d3d0/Source/DafnyLanguageServer.Test/Various/CounterExampleTest.cs)).